css/src/tokens/aspect-ratios.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * @sass-export-section="aspect-ratios"
+ */
+$aspect-ratios: (
+	'square': '1 / 1',
+	'1-2': '1 / 2',
+	'1-3': '1 / 3',
+	'2-1': '2 / 1',
+	'2-3': '2 / 3',
+	'3-1': '3 / 1',
+	'3-2': '3 / 2',
+	'4-3': '4 / 3',
+	'4-5': '4 / 5',
+	'3-4': '3 / 4',
+	'5-4': '5 / 4',
```

Comment:
@wibjorn These were added in because we already have these aspect ratios in docs-ui's `image.scss` file, with the goal of making sure people could migrate any preexisting `is-[ratio]` class to its Atlas alternative.

Reply:
Might be worth seeing if any of those classes are cullable, if they're either not in use or could be replaced with a similar enough ratio. But as it stands, I'm not sure to what degree we want some, but not all, of the ratios to be available in Atlas.

Reply:
Did an audit of the image.scss classes not being used. Left 4-3 because it's a common video ratio, and added 4-1 because I found that many of the images on the site use that ratio even though it isn't included in the image.scss file.